docs/en/zigbee/zigbee.rst

```diff
@@ -0,0 +1,159 @@
+######
+Zigbee
+######
+
+About
```

Comment:
Do you think we should mention the supported SoCs?

Reply:
@pedrominatel The Zigbee library supports all SoCs :) It can also run on ESP32, if its configured properly and have a native Zigbee SoC connected to it using UART and the device is running RCP firmware.